tools/schema-validate.py

```diff
@@ -67,6 +69,18 @@ def collect_defs(self, schema):
             else:
                 self.collect_defs(child)
 
+    def check_version(self, schema: Schema):
+        versionNumber = schema["$version"]
+        
+        majorVersion = math.floor(versionNumber / 10000)
+        minorVersion = math.floor((versionNumber % 10000) / 100)
+        patchVersion = versionNumber % 100
+        
+        versionString = f'{majorVersion}.{minorVersion}.{patchVersion}'
+        
+        if versionString not in schema["$id"]:
```

Comment:
For the validation I meant the value `schema/composition/animation.json` rather than the file name, it's to avoid inconsistencies, I'm guessing the file name will be generated by the release script

Reply:
If we update that value to be the same, existing lotties will fail validation. I think we want to allow newer schemas to validate older lotties, but that discussion is still open. The one exception may be on major version changes.

Reply:
ah ok